src/App.css

```diff
@@ -1,11 +1,36 @@
-h1 {
-  text-align: center;
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300;400;700&display=swap');
+
+body {
+  background: #fafafa;
+}
+
+a,
+button,
+select {
+  cursor: pointer;
 }
 
 .navbar {
+  padding: 1rem 5rem 0;
+  background-color: #fff;
   display: flex;
+  align-items: center;
   justify-content: space-between;
-  padding: 0 5rem;
+}
+
+.nav {
+  display: flex;
```

Comment:
- [ ] Nice job with the styling. In order to ensure we follow the DRY rule in our CSS. I will suggest you declare the highlighted property once in your CSS. Currently, it's occurring about 9x in the CSS. You can achieve this in different ways. One basic way I will show below:

At the top of your css, you can declare the flex property like so:
```
:root {
   --variable-name: flex;
}
```
And then reuse the ` --variable-name` inside the classes like so:
```
.nav {
  display: var(--flex);
 ----other properties---
  ---------
}
```